### ISSUE TITLE
Interacting with the enchanting table crashes the game

### ISSUE THREAD

**Issue by reMarce** [Globox1997/LevelZ#169]
Exit code: -1
Placing an item in any slot of the enchanting table makes the game crash and moving the mouse over the button "?" also does.
Minecraft version 1.18.2 (fabric)
Levelz 1.3.3

**Comment by Globox1997** [comment]
can you post the crash log here?

**Comment by DedicatedMedic** [comment]
I combined the related files from the 1.19 mc, 1.3.6 mod files. The crash is fixed when placing an item, the crash on hovering the helper tooltip remains. Attached are three logs from crashes. I personally have many other mods however so an incompatibility is also possible, I am simply too stubborn to remove LevelZ though as I enjoy the mod.

[2022-07-27-7.log.gz](https://github.com/Globox1997/LevelZ/files/9204861/2022-07-27-7.log.gz)
[2022-07-27-6.log.gz](https://github.com/Globox1997/Le
[2022-07-27-5.log.gz](https://github.com/Globox1997/LevelZ/files/9204880/2022-07-27-5.log.gz)
velZ/files/9204862/2022-07-27-6.log.gz)

 EDIT: Clarifying that I am also running 1.18.2 LevelZ 1.3.3. I simply opened the jar to combine the enchantment mixins from the newer version as the patch notes implied they had a fix.

**Comment by Globox1997** [comment]
Yes there is no null check in the screen class here [https://github.com/Globox1997/LevelZ/blob/78a854f161de969db7fd70e6efc87618551d3cdd/src/main/java/net/levelz/mixin/misc/EnchantmentScreenMixin.java#L40](https://github.com/Globox1997/LevelZ/blob/78a854f161de969db7fd70e6efc87618551d3cdd/src/main/java/net/levelz/mixin/misc/EnchantmentScreenMixin.java#L40)
This got fixed in a later version

**Comment by nc5432** [comment]
I can confirm that this happens no matter what item you put in

**Comment by Globox1997** [comment]
This should be fixed with the latest update

**Comment by Veluz22** [comment]
I can confirm that it is still happening
